src/constants/api.js

```diff
@@ -3,7 +3,15 @@ const API_HOST = process.env.API_HOST || 'localhost'
 const HTTP_PORT = process.env.HTTP_PORT || 3001
 export const API_HREF = `http://${API_HOST}:${HTTP_PORT}`
 export const APP_HREF = process.env.APP_HREF || 'http://localhost:3000'
-export const GRAPHQL_ENDPOINT = '/graphql'
-export const GRAPHIQL_ENDPOINT = '/graphiql'
-export const AUTH_LOGIN = '/login'
-export const AUTH_WHITELIST = [AUTH_LOGIN, GRAPHIQL_ENDPOINT]
+
+export const GRAPHQL_PATH = '/graphql'
+export const GRAPHIQL_PATH = '/graphiql'
```

Comment:
Maybe we should create a more specific constant file for routes like `@server/routes/constants` ?